docs/EN_US/ECLLanguageReference/ECLR_mods/Recrd-Index.xml

```diff
@@ -346,6 +349,91 @@ BUILD(VehicleKey2);
 
     <programlisting>NewVehicleKey := INDEX(VehicleKey,'NEW::vkey::st.city');
           //define NewVehicleKey like VehicleKey</programlisting>
+  </sect2>
+
+  <sect2 id="IDXCOMP" role="brk">
+    <title>Indexes and Compression</title>
+
+    <para><emphasis role="bold">Compression Options:</emphasis></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec align="left" colwidth="122.40pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis role="bold">LZW</emphasis></entry>
+
+            <entry>The default compression. It is a variant of the
```

Comment:
This seems to be duplicated, and should be edited in the same way, or only occur once.